app/blog/shared/BlogWrapper.js

```diff
@@ -46,34 +42,33 @@ export default function BlogWrapper({
           >
             <BaseButtonClient
               style={{ backgroundColor: '#000', color: '#ffffff' }}
-              className="py-3 px-4 mb-3 font-bold"
+              className="py-3 px-4 mb-3 ml-3 mr-3 font-bold"
```

Comment:
the buttons should not have margin left, when they get to the second row they will be an offset.
only margin right. and if you need to make it double do that.